src/components/EditForm/EditForm.tsx

```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
+import React,
+{
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../../context/TodoContext';
+import { deleteTodos, updateTodos } from '../../api/todos';
+import { Error } from '../../types/Error';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.